.github/workflows/linux.bash

```diff
@@ -64,6 +92,10 @@ case "$DISTRO" in
     dnf install -y bison ccache cmake gcc-c++ flex ninja-build \
       {boost,libedit,mariadb,ncurses,openssl,postgresql,systemd}-devel
     ;;
+
+  *)
+    echo "UNKNOWN DISTRO \"$DISTRO\"" >&2
+    ;;
```

Comment:
This would fail anyway, so it's unnecessary.

Reply:
Indeed. I put this there because during local testing I have forgotten to set the `$DISTRO` variable accordingly. I will remove this again.